astroplan/data/observatories.json

```diff
@@ -0,0 +1,277 @@
+            "Lick Observatory"
+        ], 
+        "elevation": 1290, 
+        "latitude": "37d20m36s", 
+        "longitude": "238d21m48s", 
+        "name": "Lick Observatory"
+    }, 
+    "lowell": {
+        "aliases": [
+            "Lowell Observatory", 
+            "Lowell"
+        ], 
+        "elevation": 2198, 
+        "latitude": "35d05m48s", 
+        "longitude": "248d27m54s", 
+        "name": "Lowell Observatory"
+    }, 
+    "mcdonald": {
+        "aliases": [
+            "McDonald Observatory", 
+            "mcdonald"
+        ], 
+        "elevation": 2075, 
+        "latitude": "30d40m18.0001s", 
+        "longitude": "255d58m41.9999s", 
+        "name": "McDonald Observatory"
+    }, 
+    "mdm": {
+        "aliases": [
+            "Michigan-Dartmouth-MIT Observatory", 
+            "MDM"
+        ], 
+        "elevation": 1938, 
+        "latitude": "31d57m00s", 
+        "longitude": "248d23m00s", 
+        "name": "Michigan-Dartmouth-MIT Observatory"
+    }, 
+    "mmt": {
+        "aliases": [
+            "Multiple Mirror Telescope", 
+            "MMT"
+        ], 
+        "elevation": 2608, 
+        "latitude": "31d41m18s", 
+        "longitude": "249d06m54s", 
+        "name": "Whipple Observatory"
+    }, 
+    "mso": {
+        "aliases": [
+            "MSO", 
+            "Mt. Stromlo Observatory"
+        ], 
+        "elevation": 767, 
+        "latitude": "-35d19m14.34s", 
+        "longitude": "149d01m27.6s", 
+        "name": "Mt. Stromlo Observatory"
+    }, 
+    "mtbigelow": {
+        "aliases": [
+            "mtbigelow", 
+            "Catalina Observatory"
+        ], 
+        "elevation": 2510, 
+        "latitude": "32d25m00s", 
+        "longitude": "249d16m06s", 
+        "name": "Catalina Observatory: 61 inch telescope"
+    }, 
+    "spm": {
+        "aliases": [
+            "Observatorio Astronomico Nacional, San Pedro Martir", 
+            "SPM"
+        ], 
+        "elevation": 2830, 
+        "latitude": "31d01m45s", 
+        "longitude": "244d30m47s", 
+        "name": "Observatorio Astronomico Nacional, San Pedro Martir"
+    }, 
+    "sso": {
+        "aliases": [
+            "Siding Spring Observatory", 
+            "sso"
+        ], 
+        "elevation": 1149, 
+        "latitude": "-31d16m24.1s", 
+        "longitude": "149d03m40.3s", 
+        "name": "Siding Spring Observatory"
+    }, 
+    "tona": {
+        "aliases": [
+            "tona", 
+            "Observatorio Astronomico Nacional, Tonantzintla"
+        ], 
+        "elevation": 0, 
+        "latitude": "19d01m58s", 
+        "longitude": "261d41m10s", 
+        "name": "Observatorio Astronomico Nacional, Tonantzintla"
+    }, 
+    "vbo": {
+        "aliases": [
+            "Vainu Bappu Observatory", 
+            "VBO"
+        ], 
+        "elevation": 725, 
+        "latitude": "12d34m35.976s", 
+        "longitude": "78d49m35.76s", 
+        "name": "Vainu Bappu Observatory"
+    }
+}
```

Comment:
This is standard json output – I'm generating the file with Python's json module, [see this notebook](https://github.com/bmorris3/gsoc2015/blob/master/IRAF-sites.ipynb).